packages/cli/src/bin/index.ts

```diff
@@ -70,7 +70,15 @@ program
     "Provide path to .env file to load environment variables",
   )
   .action(async (options) => {
-    dotenv.config({ path: options.envFile || [".env.local", ".env"] });
+    // Check for either envFile in options or .env file or .env.local file
+    let {error} = dotenv.config({ path: options.envFile || ".env" });
+
+    // Check if environment variables exist
+    if(error){
+      console.log(buildErrorLog("Failed to load environment variables"));
+      console.log(`${yellow("Please create .env file with necessary environment variables")} \n${cyan('touch .env && echo "AI_PROVIDER_API_KEY=your_api_key_here" > .env')}`);
+      process.exit(1);
```

Comment:
there could be cases where we don't need to read environment variables from a file. 
e.g. CI /CD flows. Such flows will break

Also in local instances, there could be cases where OPENAI_KEY is already exported.

Reply:
Yeah makes sense! I saw the `AI202` error being thrown before making any API calls, so that should be fine I guess. Fixing this!

Maybe we improve that error messages for providers, `process.env.OPENAI_API_KEY is not set` to give more context, to either create a .env file or export on terminal